stdlib/http/src/main/java/org/ballerinalang/net/http/HttpResourceDataElement.java

```diff
@@ -162,18 +162,18 @@ private String getAllowHeaderValues(HttpCarbonMessage cMsg) {
         return DispatcherUtil.concatValues(methods, false);
     }
 
-    public HttpResource validateConsumes(HttpResource resource, HttpCarbonMessage cMsg) {
+    private void validateConsumes(HttpResource resource, HttpCarbonMessage cMsg) {
```

Comment:
Shall we make the return type as boolean and return false instead of throwing an exception for validation failure